setup.md

```diff
@@ -40,44 +45,55 @@ In this workshop, we will be moving code from a Jupyter notebook into a Python p
 
 ## Installing Python Using Anaconda <a name="python_install"></a>
 
-If you already have a Python 3 verison Anaconda or MiniConda installed, you can skip this step.
+If you already have a Python 3 version Anaconda or MiniConda installed, you can skip this step.
 
-[Python][python] is a popular language for scientific computing, and great for
-general-purpose programming as well.  We recommend using Python with the conda package manager. 
-You can obtain Python and conda by either installing [Miniconda](https://docs.conda.io/en/latest/miniconda.html) (minimal installer) or [Anaconda](https://www.anaconda.com/products/individual). Anaconda comes bundled with many more python packages than miniconda. Anaconda is more user-friendly. If you are uncomfortable navigating the terminal, or are unsure of which you need, install Anaconda. Anaconda will also come bundled with the text editor Visual Studio Code, so you will not need to install a text editor in the next step.
+[Python] is a popular language for scientific computing, and great for general-purpose programming as well. 
+We recommend using Python with the conda package manager. 
+You can obtain Python and conda by either installing [Miniconda] (minimal installer) or [Anaconda]
+Anaconda comes bundled with many more Python packages than miniconda.
+Anaconda is more user-friendly.
+If you are uncomfortable navigating the terminal, or are unsure of which you need, install Anaconda.
+Anaconda will also come bundled with the text editor Visual Studio Code, so you will not need to install a text editor in the next step.
```

Comment:
```suggestion

```

Reply:
Unfortunately, I think this is no longer true or was never true! In any case, it won't hurt to remove.

Reply:
Ah, you are right. Yesterday when I was installing Anaconda on my Linux OS (inside VM) there are no VS Code. Then I thought that it is much better to start with Miniconda. Beside, Anaconda offer various package for Python 3.8, which are unusable (and have to reinstall the package that you need) when you use another Python version.